### PR TITLE
[ticket/15343] Fix forum row inner list

### PR DESCRIPTION
**Pull request by hanakin**
Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket (set the ticket ID to **your ticket ID**): PHPBB3-15343

https://tracker.phpbb.com/browse/PHPBB3-15343

### CODE REVIEW

**Comment by hanakin** [comment]
!set STYLE

**Comment by hanakin** [comment]
!set Styles

File: phpBB/styles/prosilver/theme/content.css [hunk 1]
```diff
@@ -88,6 +88,18 @@ ul.topiclist dfn {
...
+}
+
+.forum-mods {
+	display: none;
+}
+
+.forum-subs {
+	display: none;
+}
+
```
Comment (vinny) [inline phpBB/styles/prosilver/theme/content.css]: Why are you hiding this information?

**Comment by vinny** [comment]
This PR have any relation to this [topic](https://www.phpbb.com/community/viewtopic.php?f=591&t=2436676&p=14814816#p14814656)?

File: phpBB/styles/prosilver/theme/content.css (hunk 1 above)
Comment (hanakin) [reply phpBB/styles/prosilver/theme/content.css]: Sorry it was just a test it should be block I'll fix

Comment (hanakin) [reply phpBB/styles/prosilver/theme/content.css]: And yes it's related to that topic

**Comment by hanakin** [comment]
Rdy!

**Comment by hanakin** [comment]
@marc1706 not sure why I chose master for this but it should have been in 3.2.x branch so probably should cherry pick into 3.3 as it addresses an issue with layout caused by br tags in the forum rows.

**Pull request merged**